packages/transaction-controller/src/utils/simulation.ts

```diff
@@ -478,3 +560,18 @@ function getSimulationBalanceChange(
 function normalizeReturnValue(value: Hex): Hex {
   return toHex(hexToBN(value));
 }
+
+/**
+ * Get the contract interfaces for all supported tokens.
+ * @returns A map of supported tokens to their contract interfaces.
+ */
+function getContractInterfaces(): Map<SupportedToken, Interface> {
+  const supportedTokens = Object.values(SupportedToken);
+
+  return supportedTokens.reduce((acc, key) => {
+    const { abi } = SUPPORTED_TOKENS[key];
+    const contractInterface = new Interface(abi);
+    acc.set(key, contractInterface);
+    return acc;
```

Comment:
Map constructor can take an error of [key, value] pairs:
```suggestion
const getContractInterfaces: () => Map<SupportedToken, Interface> = () =>
  new Map(
    Object.values(SupportedToken).map((key) => {
      const { abi } = SUPPORTED_TOKENS[key];
      const contractInterface = new Interface(abi);
      return [key, contractInterface];
    }),
  );
```

Reply:
Done.